release_notes/ocp-4-5-release-notes.adoc

```diff
@@ -193,6 +193,19 @@ You can now filter Operators by *Infrastructure Features* in OperatorHub. For
 example, select *Disconnected* to see Operators that work in
 disconnected environments.
 
+[id="ocp-4-5-developer-perspective"]
+==== Developer Perspective
+
+You can now use the *Developer* perspective to:
+
+* Make informed decisions on installing Helm Charts in the *Developer Catalog* using the description and docs for them.
+* Uninstall, upgrade, and rollback Helm Releases.
+* Create and delete dynamic Knative event sources.
+* Deploy virtual machines, launch applications in them, or delete the virtual machines.
+* Provide Git webhooks, triggers, and  workspaces; manage credentials of private git repositories; and trouble shoot using better logs for Pipelines.
```

Comment:
s/triggers/Triggers
s/workspace/Workspace
s/trouble shoot/troubleshoot
s/Pipelines/the OpenShift Pipelines